app/views/shared/_social_media_meta_tags.html.erb

```diff
@@ -1,11 +1,11 @@
 <!-- Twitter -->
-<meta name="twitter:card" content="summary" />
-<meta name="twitter:site" content="@consul_dev" />
-<meta name="twitter:title" content="<%= social_title %>" />
-<meta name="twitter:description" content="<%= social_description %>" />
-<meta name="twitter:image" content="<%= image_url local_assigns[:twitter_image_url] || image_path_for('social_media_icon_twitter.png') %>" />
+<meta name="twitter:card" content="summary"/>
+<meta name="twitter:site" content="@consul_dev"/>
```

Comment:
Changing `@consul_dev` for `setting['twitter_handle']` would be a nice cleanup :)